.github/workflows/run-short-tests.yaml

```
@@ -1,16 +1,21 @@
-name: Run tests
+name: Run Short Tests
 
-on: [push]
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - name: Use Docker in rootless mode (To solve "Permission denied" error when removing test_app directory)
-        uses: ScribeMD/rootless-docker@0.2.2
-      - uses: actions/checkout@v2
+      # Use Docker in rootless mode (To solve "Permission denied" error when removing test_app directory)
+      - uses: ScribeMD/rootless-docker@0.2.2
```

```suggestion
      - name: Use Docker in rootless mode (To solve "Permission denied" error when removing test_app directory)
        uses: ScribeMD/rootless-docker@0.2.2
```
name is more useful, as it is printed in the log

---

Taking suggestion.